sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -2,13 +2,19 @@
 
 ## 1.0.0-beta.4 (Unreleased)
 ### Breaking Changes
+- `beginRecognizeReceipt` APIs now return a `RecognizedForm` model instead of a `RecognizedReceipt`.
 - Add `RecognizeOptions` and `RecognizeCustomFormOptions` to pass configurable options when using recognize APIs on FormRecognizerClient.
 - Rename `includeTextDetails` to `includeTextContent`
 - Change `submodels` property on `CustomFormModel` to return a `List` instead of `IterableStream`
 - Rename `fieldMap` property to `fields` on `CustomFormSubmodel` model
 - Rename `elements` property on model `FormTableCell` to `textContent`
 - Rename `includeTextDetails` refernces in parameter and model properties to `includeTextContent`
 - Remove `TextContentType` model and use `instanceOf` to detect the FormContent type
+
+### New Features
+- Added a sample to demonstrate a suggested approach for extracting information from receipts, see 
```

Comment:
do we normally add new samples to the new features section? I thought this was specifically for features added to the library